backend/ops_api/ops/resources/change_requests.py

```diff
@@ -102,64 +23,51 @@ def build_change_request_response(change_request: ChangeRequest):
         return response
 
-
-class ChangeRequestReviewAPI(BaseListAPI):
-    def __init__(self, model: ChangeRequest = ChangeRequest):
-        super().__init__(model)
-
-    def division_director_of_change_request(self) -> bool:
-        current_user_id = current_user.id
-        request_data = request.json
-        change_request_id = request_data.get("change_request_id", None)
-        if change_request_id is None:
-            return False
-        change_request: ChangeRequest = current_app.db_session.get(ChangeRequest, change_request_id)
-        if not change_request or not change_request.managing_division_id:
-            return False
-        division: Division = current_app.db_session.get(Division, change_request.managing_division_id)
-        if division is None:
-            return False
-        return (
-            division.division_director_id == current_user_id or division.deputy_division_director_id == current_user_id
-        )
-
-    @is_authorized(PermissionType.POST, Permission.CHANGE_REQUEST_REVIEW)
+    @is_authorized(PermissionType.POST, Permission.CHANGE_REQUEST)
     @jwt_required()
-    def post(self) -> Response:
-        can_update_request = self.division_director_of_change_request()
-        if not can_update_request:
-            return make_response_with_headers({}, 403)
-        request_json = request.get_json()
-        change_request_id = request_json.get("change_request_id")
-        reviewer_notes = request_json.get("reviewer_notes", None)
-        action = request_json.get("action", "").upper()
-        if action == "APPROVE":
-            status_after_review = ChangeRequestStatus.APPROVED
-        elif action == "REJECT":
-            status_after_review = ChangeRequestStatus.REJECTED
-        else:
-            raise ValueError(f"Invalid action: {action}")
-
-        reviewed_by_user_id = current_user.id
-
-        change_request = review_change_request(
-            change_request_id, status_after_review, reviewed_by_user_id, reviewer_notes
-        )
-
-        return make_response_with_headers(change_request.to_dict(), 200)
+    def patch(self) -> Response:
+        with OpsEventHandler(OpsEventType.UPDATE_CHANGE_REQUEST) as meta:
+            request_json = request.get_json()
+            change_request_id = request_json.get("change_request_id")
+
+            if not change_request_id:
+                return make_response_with_headers({"error": "change_request_id is required"}, 400)
+
+            service = ChangeRequestService(current_app.db_session)
+
+            try:
+                change_request, _ = service.update(change_request_id, request_json)
+                meta.metadata.update({"Updated Change Request": change_request.to_dict()})
+                return make_response_with_headers(change_request.to_dict(), 200)
+            except AuthorizationError:
+                return make_response_with_headers({"error": "Forbidden"}, 403)
```

Comment:
should not need to `return make_response_with_headers` in a number of places here - you can just raise the exception here and `error_handlers.py` with catch and handle it